apps/docusaurus/docs/designsystems/Komponenter/checkbox.md

```diff
@@ -36,18 +39,122 @@ Avmerkingsboksen får fokus når den velges med tastaturet.
+        <label class="detail detail--sml" for="checkbox_2">Jeg aksepterer vilkårene for...</label>
+        <div class="error__box">
+            <i class="material-symbols-outlined">exclamation</i>
+            <small>Du må huke av dette feltet</small>
+        </div>
     </div>
 </div>
+<br/>
+<br/>
 
 ```markdown
 <div class="kv-checkbox">
-    <input type="checkbox" id="checkbox_2" required/>
-    <label class="detail detail--sml" for="checkbox_2">Input tekst</label>
+    <input type="checkbox" id="checkbox_6" required/>
+    <label class="detail detail--sml" for="checkbox_6">Jeg aksepterer vilkårene for...</label>
+    <div class="error__box">
+        <i class="material-symbols-outlined">exclamation</i>
+        <small>Du må huke av dette feltet</small>
+    </div>
+</div>
+```
+
+## Eksempler
+
+### Eksempel for feilmelding
+
+Marker hvilket felt som MÅ avhukes
+
+<div class="component__display__column">
+    <div class="kv-checkbox">
+        <input type="checkbox" id="checkbox_3"/>
+        <label class="detail detail--sml" for="checkbox_3">Jeg ønsker å få tilsendt nyhetsbrevet</label>
+    </div>
+    <div class="kv-checkbox">
+        <input type="checkbox" id="checkbox_4" required/>
+        <label class="detail detail--sml" for="checkbox_4">Jeg aksepterer vilkårene</label>
+        <div class="error__box">
+        <i class="material-symbols-outlined">exclamation</i>
+        <small>Du må huke av dette feltet</small>
+        </div>
+    </div>
+</div>
+
+<br/>
+<br/>
+
+```markdown
+<div class="kv-checkbox">
+    <input type="checkbox" id="checkbox_3"/>
+    <label class="detail detail--sml" for="checkbox_3">Jeg ønsker å få tilsendt nyhetsbrevet</label>
+</div>
+<div class="kv-checkbox">
+    <input type="checkbox" id="checkbox_4"/>
+    <label class="detail detail--sml" for="checkbox_4">Jeg aksepterer vilkårene</label>
+    <div class="error__box">
+        <i class="material-symbols-outlined">exclamation</i>
+        <small>Du må huke av dette feltet</small>
+    </div>
 </div>
 ```
+
+### Eksempel for "velg minst én"
+
+"Velg minst Én" er forskjellig fra radioknapper, som lar deg bare velge en.
+
+Hvordan kommer du deg til jobb?
+
+<div class="component__display__column">
+    <div class="kv-checkbox">
+        <input type="checkbox" id="checkbox_5" required/>
+        <label class="detail detail--sml" for="checkbox_5">Til fots</label> 
+    </div> 
+    <div class="kv-checkbox">
+        <input type="checkbox" id="checkbox_6" required/>
+        <label class="detail detail--sml" for="checkbox_6">Sykkel</label> 
+    </div> 
+    <div class="kv-checkbox">
+        <input type="checkbox" id="checkbox_7" required/>
+        <label class="detail detail--sml" for="checkbox_7">Kollektivt</label> 
+    </div> 
+    <div class="kv-checkbox">
+        <input type="checkbox" id="checkbox_8" required/>
+        <label class="detail detail--sml" for="checkbox_8">Bil</label> 
+        <div class="error__box">
```

Comment:
Litt pirkete kommentar, men jeg ville flyttet denne boxen ut et hakk, slik at den kodemessig hører til hele kolonnenen, og ikke den enkelte check-boxen. Det ser helt likt ut visuelt, men gir bedre kodestruktur. Jeg har ikke testet for tilgjengelighet, men det kan godt tenkes at dette spiller en rolle for tilgjengeligheten, da riktig kodestruktur er veldig viktig for hjelpemidler som f.eks skjermleser.

Reply:
I Polen pleier vi å si at djevelen ligger i detaljene, så det er bare å være så pirkete du vil😊